packages/react-ui-core/src/Highlighter/Highlighter.js

```diff
@@ -1,51 +1,56 @@
-import React, { Component } from 'react'
+import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
 import themed from '@rentpath/react-themed'
 import classnames from 'classnames'
 import { randomId } from '@rentpath/react-ui-utils'
 
 @themed(/^Highlighter/)
-export default class Highlighter extends Component {
+export default class Highlighter extends PureComponent {
   static propTypes = {
     className: PropTypes.string,
     pattern: PropTypes.string.isRequired,
-    children: PropTypes.string,
+    ignoreCase: PropTypes.bool,
+    children: PropTypes.string.isRequired,
     theme: PropTypes.object,
     indexHighlighted: PropTypes.number,
-    text: PropTypes.string,
+    renderer: PropTypes.func,
```

Comment:
What is this needed for?

Reply:
Basically, If you look at how the highlight component is passed down from AutoSuggestField to Menu to List to ListItem, It doesn't exactly leave a lot of room for modifying how its called (in our case we need the ability to prefix an svg to the front of the list item. Given the structure of everything, it was going to involve significant refactors to how stuff is passed down. Or I could just do this. I can walk you through it and see if you have any better ideas.